js/binary/reader_test.js

```diff
@@ -681,10 +681,12 @@ describe('binaryReaderTest', function() {
     writer.writeGroup(5, dummyMessage, function() {
       writer.writeInt64(42, 42);
       writer.writeInt64(44, 44);
-      writer.writeString(43, 'The quick brown fox jumps over the lazy dog');
+      writer.writeBytes(43, [255, 255, 255, 255, 255, 255, 255, 255, 255, 255]);
```

Comment:
Please also make some comment for what this tests.

Reply:
Done

Reply:
I am a little confused about how this is failed by an invalid varint. Does ```writeInt64(43, 15)``` has the same effect? (15=8 + 7, which creates a invalid tag 7)

Reply:
Done.